src/main/java/checkout/CheckoutService.java

```diff
@@ -22,130 +21,19 @@ public CheckoutService(final Properties prop) {
-        System.out.println("/paymentMethods context:\n" + paymentMethodsRequest.toString());
-
-        PaymentMethodsResponse response = checkout.paymentMethods(paymentMethodsRequest);
-        System.out.println("/paymentMethods response:\n" + response);
-        return response;
-    }
-
-    public PaymentsResponse makePayment(PaymentsRequest paymentsRequest) throws IOException, ApiException {
-        String type = paymentsRequest.getPaymentMethod().getType();
-
-        paymentsRequest.setAmount(getAmount(type));
-        paymentsRequest.setChannel(PaymentsRequest.ChannelEnum.WEB);
-        paymentsRequest.setMerchantAccount(merchantAccount);
-
+    public CreateCheckoutSessionResponse checkoutsessions() throws IOException, ApiException {
         String orderRef = UUID.randomUUID().toString();
-        paymentsRequest.setReference(orderRef);
-        paymentsRequest.setReturnUrl("http://localhost:8080/api/handleShopperRedirect?orderRef=" + orderRef);
-
-        paymentsRequest.setShopperReference("Java Checkout Shopper");
-
-        paymentsRequest.setCountryCode("NL");
-
-        if (type.equals("alipay")) {
-            paymentsRequest.setCountryCode("CN");
-
-        } else if (type.contains("klarna")) {
-            paymentsRequest.setShopperEmail("myEmail@adyen.com");
-            paymentsRequest.setShopperLocale("en_US");
-
-            addLineItems(paymentsRequest);
-
-        } else if (type.equals("directEbanking") || type.equals("giropay")) {
-            paymentsRequest.countryCode("DE");
-
-        } else if (type.equals("dotpay")) {
-            paymentsRequest.countryCode("PL");
-            paymentsRequest.getAmount().setCurrency("PLN");
-
-        } else if (type.equals("scheme")) {
-            paymentsRequest.setOrigin("http://localhost:8080");
-            paymentsRequest.putAdditionalDataItem("allow3DS2", "true");
-            paymentsRequest.setShopperIP("0.0.0.1");
-
-        } else if (type.equals("ach") || type.equals("paypal")) {
-            paymentsRequest.countryCode("US");
-        }
-
-        System.out.println("/payments request:\n" + paymentsRequest.toString());
-
-        PaymentsResponse response = checkout.payments(paymentsRequest);
-
-        System.out.println("/payments response:\n" + response);
+        Amount amount = new Amount()
+                .currency("EUR")
+                .value(1000L);
+        CreateCheckoutSessionRequest checkoutSession = new CreateCheckoutSessionRequest();
+        checkoutSession.merchantAccount(merchantAccount);
+        checkoutSession.setChannel(CreateCheckoutSessionRequest.ChannelEnum.WEB);
+        checkoutSession.setReference(orderRef); // required
+        checkoutSession.setReturnUrl("http://localhost:8080/redirect?orderRef=" + orderRef);
+        checkoutSession.setAmount(amount);
+        CreateCheckoutSessionResponse response = checkout.sessions(checkoutSession);
+        System.out.println("/CheckoutSession response ID:\n" + response.getId());
```

Comment:
Those printouts are not very valuable are they. I'd recommend at least using logging so they can be disabled if desired. And also go through which ones to keep

Reply:
In general, we never use sout in applications, but rather loggers